projects/cdk/utils/svg/svg-linear-gradient-processor.ts

```diff
@@ -1,6 +1,8 @@
 import {SafeHtml} from '@angular/platform-browser';
 import {tuiIsString} from '@taiga-ui/cdk/utils/miscellaneous';
 
+const regexpUrlIdsStr = `url\\((("{0,1}))('*)#[a-zA-Z0-9]*('*)\\)`;
```

Comment:
<img width="606" alt="image" src="https://user-images.githubusercontent.com/12021443/228865026-a3b3a37f-05f2-43a8-83a3-58f78b598896.png">

I was expect

<img width="618" alt="image" src="https://user-images.githubusercontent.com/12021443/228865243-06d61dd5-1727-4413-8c46-7a0d5e495588.png">

Reply:
If I use the following Regexp that should fix your problem, the error goes back :/

```typescript
const regexpUrlIdsStr = `url\\((("{0,1}))('*)#(.*)('*)\\)`;
```

Reply:
I found the solution, this is the one 
```typescript
const regexpUrlIdsStr = `url\\((("{0,1}))('*)#(.*?)('*)\\)`;
```